docs/_includes/skeleton.html

```diff
@@ -1,9 +1,9 @@
 <div id="skeleton">
   <!-- {% include section-header.html name="skeleton" version=page.versions.skeleton %} -->
   <h2>@ebay/skin/skeleton <sup class="beta">BETA</sup></h2>
-  <p>A skeleton provides a simplified, graphical placeholder for content that is not yet available for the client to render; they can be considered as an alternative to the <a href="#progress-spinner">progress spinner</a> in many situations.</p>
-  <p>A skeleton is appropriate as a placeholder for content in cases where a service or action may be slow to resolve (after 500ms or so).</p>
-  <p>A skeleton is not appropriate as a placeholder for content if that content has unknown dimensions, or mismatching dimensions, which would cause unexpected page layout shift.</p>
+  <p>A skeleton is graphical placeholder, reserving physical space in the page for content that is not yet available for the client to render. A skeleton can be considered as an alternative to the <a href="#progress-spinner">progress spinner</a> in many situations.</p>
+  <p>A skeleton is appropriate as a placeholder for content in cases where a service or action may be slow to resolve.</p>
+  <p>A skeleton is not appropriate as a placeholder for content if rendering that content or placeholder would cause unexpected page layout shift (see below for more details).</p>
```

Comment:
Maybe we should be a bit more insistent here? 😄 
```suggestion
  <p>A skeleton is NOT appropriate as a placeholder for content if rendering that content or placeholder would cause unexpected page layout shift (see below for more details).</p>
```